messenger.rst

```diff
@@ -1268,6 +1268,72 @@ during a request::
         :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\KernelTestCase`
         or :class:`Symfony\\Bundle\\FrameworkBundle\\Test\\WebTestCase`.
 
+Amazon SQS
+~~~~~~~~~~
+
+.. versionadded:: 5.1
+
+    The Amazon SQS transport has been added in Symfony 5.1
```

Comment:
We'll make this change while merging, but for your information, the `versionadded` messages are standardized as: `XXX was introduced in Symfony N.N.`

Reply:
Thank you. 

I leave this unchanged then.